seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumRun.scala

```diff
@@ -171,13 +163,25 @@ private[selenium] object SeleniumRun {
   private def maybeCleanupDriver(d: WebDriver, config: SeleniumJSEnv.Config) =
     if (!config.keepAlive) d.close()
 
-  private def htmlPage(scripts: Seq[java.net.URL]): String = {
-    val scriptTags =
-      scripts.map(path => s"<script src='${path.toString}'></script>")
+  private def htmlPage(setupJsUrl: URL, input: Seq[Input], materializer: FileMaterializer): String = {
+    val setupJs = s"<script src='${setupJsUrl.toString}'></script>"
+
+    val tags = input.map {
+      case Input.Script(path) =>
+        val url = materializer.materialize(path)
+        s"<script src='${url.toString}'></script>"
```

Comment:
In the HTML builder for `testHtml` we use the `defer` attribute on the script tags (both for script and module):
https://github.com/scala-js/scala-js/blob/6398685a38a0f64bfde154bfdd5b8f0d54888ae9/test-adapter/src/main/scala/org/scalajs/testing/adapter/HTMLRunnerBuilder.scala#L68
It was necessary to guarantee the proper order of execution when there are mixed scripts and modules. I'm pretty sure it is necessary here as well.